src/Module/WireMock.php

```diff
@@ -70,4 +70,13 @@ public function receivedRequestToWireMock($builderOrCount, RequestPatternBuilder
     {
         $this->wireMock->verify($builderOrCount, $builder);
     }
+
+    /**
+     * @param RequestPatternBuilder $builder
+     * @return \WireMock\Client\LoggedRequest[]
+     */
+    public function findReceivedRequestsToWireMock(RequestPatternBuilder $builder)
```

Comment:
I think we can increase php version toward 7.1+ and add return time here. But it is not required for exactly this PR